### PR TITLE
feat(*): update webdoc

### PR DESCRIPTION
**Pull request by stepancar**
Hello!
according https://github.com/webdoc-labs/webdoc/issues/112
update webdoc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
     "npm": ">= 6"
   },
   "dependencies": {
-    "@webdoc/model": "1.1.3",
-    "@webdoc/template-library": "1.1.3",
+    "@webdoc/model": "1.2.1",
+    "@webdoc/template-library": "1.2.1",
```
Comment (bigtimebuddy) [inline package.json:43]: Can we convert these to peerDependencies instead? And maybe do a fuzzy version of `^1.2.1`?

Comment (stepancar) [reply package.json]: @bigtimebuddy , sure, updated

**Comment by stepancar** [comment]
@bigtimebuddy, this package doesn't have tests. How to check that everything works correctly?

**Comment by bigtimebuddy** [comment]
The easiest thing to do is to `npm link` or `yarn link` to a project that uses is. I know https://github.com/pixijs/filters, https://github.com/pixijs/sound, and https://github.com/pixijs/pixi.js all use it.

**Comment by stepancar** [comment]
@bigtimebuddy , I checked with pixi filters. As for me, looks good

**Comment by stepancar** [comment]
Is it possible to release new version today? 😃 , sorry for forcing

**Comment by bigtimebuddy** [comment]
I'll see what I can do. Definitely can do it by tomorrow at the latest.

**Comment by stepancar** [comment]
@bigtimebuddy , thanks!

**Pull request merged**